progressive-cactus/0.1/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM biocontainers/biocontainers:08252016-4
+
+LABEL base.image="biocontainers:08252016-4"
+LABEL version="0.1"
+LABEL software="progressiveCactus"
+LABEL software.version="08092017"
+LABEL description="multiple whole-genome alignment package"
+LABEL website="https://github.com/glennhickey/progressiveCactus"
+LABEL documentation="https://github.com/glennhickey/progressiveCactus"
+LABEL license="https://github.com/glennhickey/progressiveCactus/blob/master/LICENSE"
```

Comment:
specify the license not the file ie GPL-3.0
you could add an other LABEL license_file that would contain link to the full license file

Reply:
ok cool, i'll change it